server/controllers/auth.js

```diff
@@ -52,3 +56,62 @@ exports.login = async (req, res, next) => {
     }
   }
 };
+
+exports.registerController = async (req, res, next) => {
+  try {
+    const {
+      firstName,
+      lastName,
+      password,
+      email,
```

Comment:
During sign up, you have to determine if the artist will accept to make customized or not, so we need to send it with a request body.
```suggestion
      email,
      customized,
```